### PR TITLE
DOC: avoid downloading .tif file

### PR DESCRIPTION
**Pull request by grinsted**
Simplify by using that rasterio can read directly from http. 

This also removes imports of requests and os.

### CODE REVIEW

**Comment by fmaussion** [comment]
Thanks! Can you do the same for the other example as well?

**Comment by grinsted** [comment]
Done.

**Comment by fmaussion** [comment]
@grinsted what OS are you running this on? I'm a bit concerned about this here: https://github.com/mapbox/rasterio#ssl-certs

I came into this while trying it locally, and even after setting the env variable I still get this error:

```
RasterioIOError: '/vsicurl/https://github.com/mapbox/rasterio/raw/master/tests/data/RGB.byte.tif' does not exist in the file system, and is not recognized as a supported dataset name.
```

I wonder if I'm the only one in that case, or if other users might be affected as well

**Comment by grinsted** [comment]
I wrote this code on Windows 10. I also use the rasterio feature on a Linux
box (but not via xarray). I use anaconda on both machines.

**Comment by grinsted** [comment]
I have just tested on a linux box and it works. I have access to a geo-notebook on a cloud service via my university . sys.version says:

3.6.7 | packaged by conda-forge | (default, Feb 28 2019, 09:07:38) 
[GCC 7.3.0]

**Comment by fmaussion** [comment]
Thanks! Yes, I've tested on a conda env and it works. So my take is that it probably won't work out-of-the box on debian/ubuntu/mint where rasterio is installed via pip.

So my question to @shoyer or @jhamman : should we change this example for the nicer code above, even if a minority of people (pip users like me) might find that this doesn't work for them?

**Comment by grinsted** [comment]
I wonder why it desn't work with pip. It is a feature inherited from [GDAL](https://gdal.org/gdal_virtual_file_systems.html). I guess it has something to do with how that is compiled.

**Comment by fmaussion** [comment]
> I wonder why it desn't work with pip.

Because of that: https://github.com/mapbox/rasterio#ssl-certs . It will work, but not out-of-the box. I personally don't mind, but we might get a few "your example doesn't work" requests here although it has nothing to do with xarray, this is why I'm asking others devs before merging.

**Comment by dcherian** [comment]
How about adding a comment saying that this may/may not work and suggest a workaround

**Comment by fmaussion** [comment]
> How about adding a comment saying that this may/may not work and suggest a workaround

Thanks for the suggestion! I've decided against it because it will be too wordy - let's see if this is really affecting that many people first.

Thanks @grinsted for the PR!
